### DISCUSSION TITLE
What is IP_ADDRESS_SALT?

### DISCUSSION THREAD

**Discussion by nelsonlaidev** [theodorusclarence/theodorusclarence.com#317] [Q&A] [answered]
I refer to several blogs with "like" buttons. They all have the IP_ADDRESS_SALT environment variable, but I don't know what should be filled in.

I hope someone can help me. Thanks.

**Comment by theodorusclarence** [comment, marked as answer]
Hi, it's for this function

https://github.com/theodorusclarence/theodorusclarence.com/blob/af9b733304f0fef705753239bc93f8f083612700/src/lib/helper.server.ts#L1-L18

for salt, you can use any string you like. I usually use a passphrase https://www.useapassphrase.com/

for example:
```
IP_ADDRESS_SALT="tubular-grief-small-darkish"
```

**Reply by nelsonlaidev** [reply]
Thanks for explaining ❤️
